plugins/modules/win_ntp.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: 2020, Sakar Mehra (@sakar97) <sakar.mehra@gslab.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_ntp
+short_description: Manage Windows NTP Server
+author: Sakar Mehra (@sakar97)
+requirements:
+  - This module requires Windows Server 2012R2 or Newer
+description:
+  - Manage Windows NTP Client
+  - Adds, Removes and Modifies NTP Server in windows machine.
+options:
+  peerlist:
+    description:
+      - Fully qualified domain name or IP address from which computer obtain timestamp.
+    type: list
```

Comment:
Should also add `elements: str` to state it's a list of strings that are accepted.